sherpa/optmethods/__init__.py

```diff
@@ -372,8 +372,8 @@ class LevMar(OptMethod):
        free parameters.
     epsfcn : number
        This is used in determining a suitable step length for the
-       forward-difference approximation; default is sqrt( DBL_EPSILON
-       ) ~ 1.19209289551e-07, where DBL_EPSILON is the smallest number
+       forward-difference approximation; default is FLT_EPSILON ~ 1.19209289551e-07,
+       where DBL_EPSILON is the smallest number
```

Comment:
I'm targeting a "nice" display of the docstring in an 80-column terminal, so ideally docstrings shouldn't be too long. It is obviously less-than ideal for heavily indented docstrings, so it's not a hard limit. When editing an existing docstring I tend to just try and go with the current line-length limits, unless I'm willing to re-edit the whole thing.